marketplaceItems/voiceScopeApp/appResources/appData/voiceScope_app.js

```diff
@@ -41,7 +41,7 @@
             case "EVENT_BRIDGE_OPEN_MESSAGE":
                 ui.sendToHtml({
                     type: "buttonStatus",
-                    value: enabled
+                    value: Settings.getValue("voiceScopeEnabled")
```

Comment:
please add a default value for this Settings key for when it doesn't exist, i.e.:
`Settings.getValue("voiceScopeEnabled", false);`